### ISSUE TITLE
gigasecond: Unclear why `test6` should be `assertNotEquals`

### ISSUE THREAD

**Issue by tstirrat15** [exercism/xphp#98]
My solution fails `test6` with the following output, after having passed all of the other tests.

Output from the test:
```php
<snip>
There were 2 failures:

1) GigasecondTest::test6
Failed asserting that DateTime Object &000000005023551d000000007ab41f16 (
    'date' => '2046-10-02 01:46:40.000000'
    'timezone_type' => 3
    'timezone' => 'UTC'
) is not equal to DateTime Object &000000005023551d000000007ab41f16 (
    'date' => '2046-10-02 01:46:40.000000'
    'timezone_type' => 3
    'timezone' => 'UTC'
).
</snip>
```

This is the only test that has a naked date, as opposed to a datetime. I can't tell whether this is expected behavior, and if so, why?

Any help would be appreciated.

**Comment by brettsantore** [comment]
The test is making sure your `from` implementation is not modifying and returning the original object, or it is not [passed by reference](https://blog.penjee.com/wp-content/uploads/2015/02/pass-by-reference-vs-pass-by-value-animation.gif). It would appear that this is your issue.

**Comment by tstirrat15** [comment]
Ah, okay. Is there anything in the test framework that prevents the test name from indicating this?

**Comment by petemcfarlane** [comment]
Oh dear @tstirrat15 I'm afraid the test names for this exercise are quite confusing. If you'd like to submit a PR to fix them you would be very welcome! If not I'll try to get round to fixing this this week. Thanks for raising this issue!

**Comment by tstirrat15** [comment]
@brettsantore: No worries. I figured it out pretty quickly after that. I just wasn't clear on what the test was testing for. I'm also happy to submit a PR for this one - I just haven't had time in the past couple of weeks.

**Comment by tstirrat15** [comment]
PR submitted. I didn't know about using the `fixes` keyword in the commit message, so it isn't linked, but it's there.

**Comment by petemcfarlane** [comment]
Thanks! Fixed in #103
